### ISSUE TITLE
Only Display for Admins

### ISSUE THREAD

**Issue by mahodder** [explodybits/hookr-plugin#5]
Sorry if I missed it but it would be awesome if, by default, hooks we're only displayed to logged in admin users.

Great plugin!

**Comment by chromeo** [comment]
I agree, even if I turn off public display in the options panel there are still side effects showing on the public side (like breaking the handlebars.js parser).

**Comment by explodybits** [comment]
Everyone, please migrate over to using
https://wordpress.org/plugins/wp-inspect/

I've already fixed numerous buts and made several enhancements. Also, I have
a lot of features planned.

**Comment by explodybits** [comment]
Are these because it is running via an ajax call?
